src/backend/storeManagers/gog/library.ts

```diff
@@ -377,6 +377,8 @@ export async function getInstallInfo(
   installPlatform = 'windows',
   lang = 'en-US'
 ): Promise<GogInstallInfo | undefined> {
+  installPlatform = installPlatform.toLowerCase()
```

Comment:
```suggestion
  installPlatform = installPlatform === 'Mac' ? 'osx' : installPlatform.toLowerCase()
```

this fixes mac as well